client/grpc/oracle/proof/proof.pb.go

```diff
@@ -0,0 +1,5685 @@
+	NextValidatorHashAndConsensusHash []byte `protobuf:"bytes,6,opt,name=next_validator_hash_and_consensus_hash,json=nextValidatorHashAndConsensusHash,proto3" json:"next_validator_hash_and_consensus_hash,omitempty"`
+	LastResultsHash                   []byte `protobuf:"bytes,7,opt,name=last_results_hash,json=lastResultsHash,proto3" json:"last_results_hash,omitempty"`
+	EvidenceAndProposerHash           []byte `protobuf:"bytes,8,opt,name=evidence_and_proposer_hash,json=evidenceAndProposerHash,proto3" json:"evidence_and_proposer_hash,omitempty"`
+}
+
+func (m *BlockHeaderMerkleParts) Reset()         { *m = BlockHeaderMerkleParts{} }
+func (m *BlockHeaderMerkleParts) String() string { return proto.CompactTextString(m) }
+func (*BlockHeaderMerkleParts) ProtoMessage()    {}
+func (*BlockHeaderMerkleParts) Descriptor() ([]byte, []int) {
+	return fileDescriptor_ae72765f9c902173, []int{16}
+}
+func (m *BlockHeaderMerkleParts) XXX_Unmarshal(b []byte) error {
+	return m.Unmarshal(b)
+}
+func (m *BlockHeaderMerkleParts) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	if deterministic {
+		return xxx_messageInfo_BlockHeaderMerkleParts.Marshal(b, m, deterministic)
+	} else {
+		b = b[:cap(b)]
+		n, err := m.MarshalToSizedBuffer(b)
+		if err != nil {
+			return nil, err
+		}
+		return b[:n], nil
+	}
+}
+func (m *BlockHeaderMerkleParts) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_BlockHeaderMerkleParts.Merge(m, src)
+}
+func (m *BlockHeaderMerkleParts) XXX_Size() int {
+	return m.Size()
+}
+func (m *BlockHeaderMerkleParts) XXX_DiscardUnknown() {
+	xxx_messageInfo_BlockHeaderMerkleParts.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_BlockHeaderMerkleParts proto.InternalMessageInfo
+
+func (m *BlockHeaderMerkleParts) GetVersionAndChainIdHash() []byte {
+	if m != nil {
+		return m.VersionAndChainIdHash
+	}
+	return nil
+}
+
+func (m *BlockHeaderMerkleParts) GetHeight() uint64 {
+	if m != nil {
+		return m.Height
+	}
+	return 0
+}
+
+func (m *BlockHeaderMerkleParts) GetTimeSecond() uint64 {
+	if m != nil {
+		return m.TimeSecond
+	}
+	return 0
+}
+
+func (m *BlockHeaderMerkleParts) GetTimeNanoSecond() uint32 {
+	if m != nil {
+		return m.TimeNanoSecond
+	}
+	return 0
+}
+
+func (m *BlockHeaderMerkleParts) GetLastBlockIdAndOther() []byte {
+	if m != nil {
+		return m.LastBlockIdAndOther
+	}
+	return nil
+}
+
+func (m *BlockHeaderMerkleParts) GetNextValidatorHashAndConsensusHash() []byte {
+	if m != nil {
+		return m.NextValidatorHashAndConsensusHash
+	}
+	return nil
+}
+
+func (m *BlockHeaderMerkleParts) GetLastResultsHash() []byte {
+	if m != nil {
+		return m.LastResultsHash
+	}
+	return nil
+}
+
+func (m *BlockHeaderMerkleParts) GetEvidenceAndProposerHash() []byte {
+	if m != nil {
+		return m.EvidenceAndProposerHash
+	}
+	return nil
+}
+
+// CommonEncodedVotePart represents the common part of encoded vote
+type CommonEncodedVotePart struct {
```

Comment:
We used `hexBytes` to make json encode as hex if you use `[]byte` it will encode with base 64